src/Communication.cs

```diff
@@ -31,42 +38,40 @@ public class UserParameter
         public string BookmarkId { get; set; }
         public bool OnDemand { get; set; }
         public string PrivateKeyPath { get; set; }
+        public int CleanupTimeout { get; set; } = 20000;
```

Comment:
do you handle that internal in miliseconds?

Reply:
@konne 
Yes, i handle the time out in milliseconds.
If the property is set, I calculate x 1000